### PR TITLE
Add Kernel#raise vs E2MM#Raise

### PR DESCRIPTION
**Pull request by sshaw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,41 @@ Comparison:
 module_eval with string:     1129.7 i/s - 1.19x slower
 ```
 
+##### `raise` vs `E2MM#Raise` for raising (and defining) exeptions  [code](code/general/raise-vs-e2mmap.rb)
```
Comment (JuanitoFatas) [inline README.md]: Could you add some explanations here? :bow:

File: README.md [hunk 2]
```diff
@@ -154,6 +154,44 @@ Comparison:
...
+
+Ruby's [Exception2MessageMapper module](http://ruby-doc.org/stdlib-2.2.0/libdoc/e2mmap/rdoc/index.html)
+allows one to define and raise exceptions with predefined messages.
+
```
Comment (sshaw) [inline README.md]: Is this too terse?

**Comment by winston** [comment]
Interesting! I didn't even know about e2mm. :bow:

**Comment by JuanitoFatas** [comment]
@sshaw Thanks for the Pull Request! @sshaw Could you share where you / which project that uses E2MM?

**Comment by sshaw** [comment]
Heh, this may be the only project to ever use it :sunglasses:

**Pull request closed without merging**